CDCgov/NEDSS-Modernization
# Showing Pagination while reloading the page everytime

## Description

Showing Pagination while reloading the page every time.
Changed Add New button color in Patient Search to make it look disabled.
Profile - Header needs more padding on left
Search - line border colors should match #DCDEE0

## Tickets

* https://cdc-nbs.atlassian.net/browse/CNFT1-1582
* https://cdc-nbs.atlassian.net/browse/CNFT1-1590
* https://cdc-nbs.atlassian.net/browse/CNFT1-1592
* https://cdc-nbs.atlassian.net/browse/CNFT1-1594

## Steps to verify

1. Patient Search- Showing Pagination while reloading the page every time.
2. Patient Search- Changed Add New button color in Patient Search to make it look disabled.
3. Patient Profile - Added more padding on left to the Header
4. Patient Search - line border colors should match #DCDEE0
